cms/static/js/certificates/models/certificate.js

```diff
@@ -0,0 +1,71 @@
+// Backbone.js Application Model: Certificate
+
+define([
+    'backbone', 'js/certificates/models/signatory', 'js/certificates/collections/signatories', 'underscore', 'underscore.string', 'gettext', 'backbone-relational', 'backbone.associations', 'coffee/src/main'
+],
+function(Backbone, Signatory, SignatoryCollection, _, str, gettext) {
+    'use strict';
+    _.str = str;
```

Comment:
underscore.string is reflected at this path  js/vendor/underscore.string.min , 
you can see the mapping in require-config.js as well.